test-data/genops-set.test

```diff
@@ -15,7 +15,7 @@ def f():
     r8 :: object
     r9 :: bool
 L0:
-    r0 = set
+    r0 = set 
```

Comment:
Too bad all the changes to this file are *adding* trailing whitespace. :-(

Reply:
#484 fixes the issue that can cause trailing whitespace to creep in, and I'll remove it from this.